ska_sun/tests/test_sun.py

```diff
@@ -1,18 +1,18 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
 import numpy as np
-import pytest
 from Quaternion import Quat
 
-from ska_sun.sun import (
+import ska_sun
+from ska_sun import (
     allowed_rolldev,
     apply_sun_pitch_yaw,
     get_sun_pitch_yaw,
     nominal_roll,
     off_nominal_roll,
 )
-from ska_sun.sun import pitch as sun_pitch
-from ska_sun.sun import position
+from ska_sun import pitch as sun_pitch
+from ska_sun import position
```

Comment:
I think there is a bug in `black` and it refuses to allow putting `position` into the long import list above. Or maybe it only wants 5 things in an import, who knows.